### PR TITLE
functional-tester: less intensive stresser

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/issues/4634.

### CODE REVIEW

**Comment by xiang90** [comment]
Let's run this for a while before merging.

**Comment by gyuho** [comment]
@xiang90 Yeah, I have to keep running my test branch anyway because of the boltdb patch.
Thanks.

**Comment by gyuho** [comment]
@xiang90 Any objection? Been running over 3000 rounds, with no inconsistent hash error.
Deploying master branch is just being blocked on boltdb issue.

File: tools/functional-tester/etcd-tester/cluster.go [hunk 1]
```diff
@@ -129,6 +129,9 @@ func (c *cluster) Bootstrap() error {
 		}
 	}
 
+	// TODO: Too intensive stresser panics etcd node with
```
Comment (xiang90) [inline tools/functional-tester/etcd-tester/cluster.go]: panic etcd member

**Comment by xiang90** [comment]
LGTM

File: tools/functional-tester/etcd-tester/cluster.go (hunk 1 above)
Comment (gyuho) [inline tools/functional-tester/etcd-tester/cluster.go]: Fixed. Thanks!

**Pull request merged**